### ISSUE TITLE
NTR: gamma-glutamylphenylalanine present in urine (Lab: chemistry: amino compounds)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8504]
New term request gamma-glutamylphenylalanine present in urine (segal_030502164331):

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000281
- label: Elevated urinary gamma-glutamylphenylalanine level
- parents: [Increased urinary non-proteinogenic amino acid level (HP:0033098)](https://hpo.jax.org/app/browse/term/HP:0033098)
- pmid: PMID:466810
- def: The amount of gamma-glutamylphenylalanine level in the urine, when normalized for urine concentration, is above the upper limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8504
- synonyms:
